pkg/datapath/iptables/iptables.go

```diff
@@ -281,13 +314,15 @@ type Manager struct {
 }
 
 type reconcilerParams struct {
-	clock          clock.WithTicker
-	localNodeStore *node.LocalNodeStore
-	db             *statedb.DB
-	devices        statedb.Table[*tables.Device]
-	proxies        chan reconciliationRequest[proxyInfo]
-	addNoTrackPod  chan reconciliationRequest[noTrackPodInfo]
-	delNoTrackPod  chan reconciliationRequest[noTrackPodInfo]
+	clock                  clock.WithTicker
+	localNodeStore         *node.LocalNodeStore
+	db                     *statedb.DB
+	devices                statedb.Table[*tables.Device]
+	proxies                chan reconciliationRequest[proxyInfo]
+	addNoTrackPod          chan reconciliationRequest[noTrackPodInfo]
+	delNoTrackPod          chan reconciliationRequest[noTrackPodInfo]
+	setNoTrackHostPorts    chan reconciliationRequest[noTrackHostPortsPodInfo]
+	removeNoTrackHostPorts chan reconciliationRequest[podAndNameSpace]
```

Comment:
just nit: should we name it consistently `addNoTrackHostPorts`, `delNoTrackHostPorts`?

Also for the commit descriptions (e.g. https://github.com/cilium/cilium/pull/41364/commits/e5090ce719e21ab969140f0f28a8c233c92547f9), please format them in similar style with line break between 70-80 chars. See Cilium (or Linux kernel) git log for examples.